save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -0,0 +1,32 @@
+package org.cqfn.save.backend.service
+
+import org.cqfn.save.backend.repository.TestExecutionRepository
+import org.cqfn.save.entities.TestExecution
+import org.springframework.stereotype.Service
+
+/**
+ * Service for test result
+ */
+@Service
+class TestExecutionService(private val testResultRepository: TestExecutionRepository) {
+    /**
+     * @param testExecutions list of test executions
+     */
+    fun saveTestResult(testExecutions: List<TestExecution>) {
+        testExecutions.forEach { testExec ->
+            testResultRepository.run {
+                val foundTestExec = this.findById(testExec.id)
+                 if (!foundTestExec.isPresent) {
+                     println("error")
+                 } else {
+                     foundTestExec.map {
```

Comment:
Is `foundTestExec` a collection? I think in norma situation in would be a collection with a single element, so if there are more you shouldn't update all of them